src/lesson4.java

```diff
@@ -0,0 +1,202 @@
+import java.util.Map;
+import java.util.Random;
+import java.util.Scanner;
+
+public class lesson4 {
+    static final int SizeMap = 5;
+    static final int WinItems = 5;
+    static final char DotX = 'X';
+    static final char DotO = 'O';
+    static final char DotEmpty = '*';
+    static char[][] GameMap = new char[SizeMap][SizeMap];
+
+    //fill map with '*"
+    static void Map() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.